.github/workflows/nightly.yml

```diff
@@ -45,3 +51,59 @@ jobs:
 
     - name: Run tests under valgrind
       run: ${{github.workspace}}/test/test_valgrind.sh ${{github.workspace}} ${{github.workspace}}/build ${{matrix.tool}}
+
+  coverity:
```

Comment:
for the test purpose, please enable this workflow for PR trigger (we'll have to see if everything works fine on this repo, incl. secret var. setup)

Reply:
it won't work on PR, I'm afraid, you can remove the PR trigger, YOLO 😉